test/test_helper.exs

```diff
@@ -1,4 +1,4 @@
-ExUnit.start()
```

Comment:
This is for us to not pollute the `stdout` when running tests because of the added Logger.info calls.

Reply:
It will also hide the `Logger.warning` calls from extra keys we are not handling, which is not bad.